app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/CustomScriptRow/index.js

```diff
@@ -47,8 +133,66 @@ const CustomScriptRow = ({ rowIndex, pairIndex, isLastIndex, deleteScript, scrip
   };
 
   const renderURLInput = () => {
+    const renderCodeEditorForScriptAttributes = () => {
+      const handleScriptChange = (value) => {
+        /* THIS IS A TEMPORARY RULE ATTRIBUTE */
+        dispatch(
+          actions.updateRulePairAtGivenPath({
+            pairIndex,
+            updates: {
+              [`scripts[${scriptIndex}].wrapperElement`]: value,
+            },
+          })
+        );
+      };
+
+      const handleCodeFormattedFlag = () => {
+        setIsCodeFormatted(true);
+        setTimeout(() => {
+          setIsCodeFormatted(false);
+        }, 2000);
+      };
+      return (
+        <Col span={24} data-tour-id="code-editor">
+          <Row
+            key={rowIndex}
+            span={24}
+            style={{
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            <Col xl="12" span={24}>
+              <CodeEditor
+                id={pair.id}
+                height={75}
```

Comment:
@RuntimeTerror10 @rohanmathur91 Do we specify fixed values like this?

Reply:
Yes, the code editor height is resizable by dragging so we need to specify a fixed minimum height for the code editor.

Reply:
Yes, the code editor height is resizable by dragging so we need to specify a fixed minimum height for the code editor.